internal/controllers/machine/drain/filters.go

```diff
@@ -242,3 +251,82 @@ func (d *Helper) skipDeletedFilter(_ context.Context, pod *corev1.Pod) PodDelete
 	}
 	return MakePodDeleteStatusOkay()
 }
+
+func (d *Helper) drainLabelFilter(_ context.Context, pod *corev1.Pod) PodDeleteStatus {
+	if labelValue, found := pod.ObjectMeta.Labels[clusterv1.PodDrainLabel]; found && labelValue == "skip" {
```

Comment:
q: should we make the string comparison case insensitive (I don't if there are conventions)